libnet/configure.ac

```diff
@@ -4,25 +4,50 @@ dnl All rights reserved.
 dnl
 dnl Process this file with autoconf to produce a configure script.
 
-AC_INIT([libnet],[1.2-rc3])
+AC_INIT([libnet],[1.2-rc3],,,[http://github.com/sam-github/libnet])
 AC_MSG_RESULT(beginning autoconfiguration process for libnet-${PACKAGE_VERSION} ...)
 AC_CANONICAL_TARGET
 AC_CONFIG_SRCDIR([src/libnet_build_ip.c])
 AC_CONFIG_MACRO_DIR([m4])
-AM_INIT_AUTOMAKE
+AM_INIT_AUTOMAKE([foreign subdir-objects no-dependencies -Wall])
 AC_CONFIG_HEADERS([include/config.h])
-AM_MAINTAINER_MODE
 
 dnl
 dnl Check for the usual programs
 dnl
 AC_PROG_CC
 AC_PROG_INSTALL
 AC_USE_SYSTEM_EXTENSIONS
+AX_CFLAGS_WARN_ALL
```

Comment:
Yeah I saw that you added it to the m4 folder. I'm afraid that's not enough. It seems that it depends on another macro that should be available on the system.

I guess the reason it doesn't complain for you is because you have the entire autoconf-archive in your path (through apt maybe?). I don't. I jus cloned their repo and have that sitting somewhere outside my path.

I'll try again later tonight. In general, we have to include all m4 macros this one depends on, drop it, or add the autoconf-archive as a dependency. I'm not a big fan the latter...